raml-maven-plugin/src/test/java/uk/gov/justice/raml/maven/lintchecker/rules/LintCheckMojoTest.java

```diff
@@ -18,17 +20,17 @@
     public void testFindAllMojosUnderTheSourceDirectory() throws Exception {
 
         final File pom = getTestFile("src/test/resources/lint-check/pom.xml");
-        final LintCheckMojo mojo = (LintCheckMojo) lookupConfiguredMojo(pom, "lint-check");
+        final LintCheckMojo mojo = (LintCheckMojo) lookupConfiguredMojo(pom, "raml-lint-check");
 
-        assertThat(mojo.rules.size(), is(2));
-        assertThat(mojo.rules.get(0), is(instanceOf(SucceedingLintCheckRule.class)));
-        assertThat(mojo.rules.get(1), is(instanceOf(FailingLintCheckRule.class)));
+        assertThat(mojo.rules().size(), is(2));
```

Comment:
We should be asserting that each of the rules got executed, not that the rules are loaded. Then. we wouldn't need the protected method to get the rules on the Mojo.